documentation/getting_started/bug_bounty.md

```diff
@@ -1,74 +1,83 @@
 
-Security of the platform is our highest priority. 
+Security of the platform is our highest priority.
 All contract code and balances are publicly verifiable, and security researchers are eligible for a bug bounty for reporting undiscovered vulnerabilities.
 
 ## Audit
-<!-- TODO: Add details of audit. 
-The contracts have been carefully audited by smart contract security experts [names]. 
+
+<!-- TODO: Add details of audit.
+The contracts have been carefully audited by smart contract security experts [names].
 The audit report can be found [here].
---> 
+-->
 
 UMA encourages the community to audit our contracts and security; we also encourage the responsible disclosure of any issues.
 The bug bounty program is intended to recognize the value of working with the community of independent security researchers, and sets out our definition of good faith in the context of finding and reporting vulnerabilities, as well as what you can expect from us in return.
 
 ## Rewards
-UMA offers substantial rewards for discoveries that can prevent the loss of assets, the freezing of assets, or harm to users. 
 
-Rewards range from $500 to $25,000. 
-All rewards will be paid in ETH, and the amount of compensation will vary depending on severity. 
+UMA offers substantial rewards for discoveries that can prevent the loss of assets, the freezing of assets, or harm to users.
+
+Rewards range from $500 to $25,000.
+All rewards will be paid in ETH, and the amount of compensation will vary depending on severity.
 The severity is calculated according to the [OWASP](https://owasp.org/www-project-risk-assessment-framework/) risk rating model based on Impact and Likelihood.
 
 ![](severity.png)
 
 ## Scope
-The scope of our bug bounty program includes core smart contracts related to release v1.0 of the DVM and financial templates. 
-It does not include known issues with the intended behavior. 
-Our [documentation site](https://docs.umaproject.org/uma/index.html) has an extensive overview of the intended behavior of the system. 
+
+The scope of our bug bounty program includes core smart contracts related to release v1.0 of the DVM and financial templates.
+It does not include known issues with the intended behavior.
+Our [documentation site](https://docs.umaproject.org/uma/index.html) has an extensive overview of the intended behavior of the system.
 
 ### In scope:
-* DVM contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/oracle/implementation)
-<!-- TODO: add token contracts when ready
-* Multi-sponsor token contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates/implementation)
--->
+
+- DVM contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/oracle/implementation)
+  <!-- TODO: add token contracts when ready
+- Multi-sponsor token contracts [link](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates/implementation)
+  -->
```

Comment:
It looks like the hyphen as bullet is causing an additional arrow to show up at the end of this section. 
<img width="306" alt="Screen Shot 2020-04-17 at 10 43 30 AM" src="https://user-images.githubusercontent.com/44578494/79581341-4f2db000-8098-11ea-87e3-b0b35ee12c8c.png">